benchmarks/fluentd/benchmark.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'harness'
+
+Dir.chdir(__dir__)
+use_gemfile
+
+require 'fluent/engine'
+require 'fluent/parser'
+
+# Prepare a fixture
+ltsv = 1000000.times.map do
+  "time:2022-08-07 07:38:31,842	module:main.py	level:DEBUG	message:No kernel command line url found.\n"
+end.join
```

Comment:
Can you add a bit of variety to the messages? Are there different types of log messages?

Reply:
Looks like this is an error message from `cloud-init`. I used some error messages in its [main.py](https://github.com/number5/cloud-init/blob/a31370afe3887be67bfa2b499e102bcc10814447/cloudinit/cmd/main.py) at https://github.com/Shopify/yjit-bench/pull/243/commits/38d4d0fb49d599d8a8d64d4f9583d48169946138.